docset/windows/nettcpip/set-nettcpsetting.md

```diff
@@ -57,13 +57,16 @@ The **Set-NetTCPSetting** cmdlet modifies a TCP setting.
 TCP settings are optimized for different network conditions including latency and congestion.
 To apply a TCP setting to a port number or destination IP address range, create a transport filter by using the New-NetTransportFilter cmdlet.
 
-You can only modify the Custom TCP setting.
+> [!IMPORTANT] 
```

Comment:
We cannot use the tags like IMPORTANT because this renders in the terminal and those tags are for markdown. Please remove.

Reply:
Sorry, I wasn't clear. We cannot use any tags that render. It just has to be text.